### PR TITLE
adds yieldies info

### PR DESCRIPTION
**Pull request by 0xean**

### CODE REVIEW

File: yieldies/fees.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Fees
+
+There are a few places that fees are involved in Yieldies that user's should be aware of.
```
Comment (willyogo) [inline yieldies/fees.md:3]: user's -> users

**Review by 0xdef1cafe — APPROVED** [review]

File: yieldies/fees.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Fees
+
+There are a few places that fees are involved in Yieldies that user's should be aware of. 
+
+1. Performance Fees - the Shapeshift DAO, as the administrator of Yieldies, has the ability to 
+levy a performance fee on the yield generated by the underlying assets.  The fee amount can be set
+by the Treasury Management and Diversification Committee.
+2. When Instant Un-staking using the Liquidity Reserve - A fee can be charged to compensate liquidity providers in the Liquidity Reserve. The fee amount can be set
+by the Treasury Management and Diversification Committee.
+3. When Instant Un-staking using Curve - Only Curve fee's apply, there is no additional fees taken by Yieldies.
```
Comment (willyogo) [inline yieldies/fees.md:10]: Only Curve fee's apply -> Only Curve's fees apply

File: yieldies/unstaking.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Currently, yieldies require a user to first request withdrawal of their assets prior to being able to redeem.  This is due to the fact that the underlying
+assets are aren't immediately liquid and are being used in DeFi strategies to generate yield. Once the request is made the funds will become available for withdrawal
+within 7-14 days.  
+
+There are other options however if you require immediate liquidity to the underlying assets.
+
+#### If you require immediate liquidity your options may include:
+
+- **Instant un-staking through the Liquidity Reserve** - a Liquidity Reserve is available to for each Yieldy and its associated
```
Comment (willyogo) [inline yieldies/unstaking.md:14]: remove either "to" or "for"

File: yieldies/unstaking.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+underlying asset. If there is sufficient liquidity in the reserve a user may swap their Yieldy for the underlying asset instantly but incur a fee in doing so. 
+This fee is distributed back to the liquidity providers who have supplied the underlying asset to the pool.
+
+- **Instant un-staking through a Curve pool** - Currently all Yieldies will leverage Tokemak for yield generation. The means that the underlying assets are deposited
+to Tokemak and in turn the Staking contract is issued tAssets.  If a Curve pool for these tAssets exists, the Staking contract can facilitate instant un-staking
+by swapping the tAsset for the underlying asset at the current market rate, returning the underlying asset to the user.  In this case, no fee is charged by Yieldies
+only by the curve pool used for the swap.
```
Comment (willyogo) [inline yieldies/unstaking.md:21]: only by the curve pool used for the swap -> , and instead the swap is subject to fees and slippage on the Curve pool.

**Review by willyogo — APPROVED** [review]
looks great!

**Pull request merged**